guidelines/terms/20/synchronized-media.html

```diff
@@ -5,4 +5,5 @@
       interactive components, unless the media is a <a>media alternative for text</a> that is clearly labeled as such
    </p>
    
+   <p class="note">"Synchronized" refers to all audio and video content intended to be played according to the same time base (i.e., in the same video file or multimedia experience). Images and sounds that are out of synch with each other (intentionally or unintentionally) are included by the definition. Asynchronous sound, for example, is still being experienced in the context of time-based, synchronized media.</p>
```

Comment:
"according to the same time base" sounds overly convoluted/impenetrable. perhaps something simpler, like "at the same time", "in parallel", or similar?

I think we've been trying to avoid "e.g.", "i.e.", and so on for simpler language. maybe replace here with "that is" or "for instance"?

maybe expand a bit what's mean by "Asynchronous sound", again using simpler language? Maybe a bit wordy, but something like "For instance, even if a video only includes 'incidental' music that does not explicitly synchronise with any of the visuals, ..."